spec/system/folder_action_system_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'FolderAction', type: :system, js: true do
+  include SystemHelpers
+
+  # Run before each testcase
+  before(:each) do
+    login_as_user(:admin)
+    @encryptable = Encryptable.first
+    @folder = Folder.find(@encryptable.folder_id)
+    @team = Team.find(@folder.team_id)
+  end
+
+  # can select folder and team from side-navbar
+  it 'opens folder and team in side-navbar' do
```

Comment:
Now about our discussion from Tuesday about necessary or non-necessary comments :joy: 
You could just name the test case as to make the comment unnecessary e.g. `it "can select folders and teams from sidebar"`
However, I digress and there is no need to spend more time on such small things for a test case :grin: 
Everything else looks good to me, good job!